server/src/test/java/com/octopus/teamcity/opentelemetry/server/TeamCityBuildListenerTest.java

```diff
@@ -32,13 +34,10 @@ void setUp(@Mock EventDispatcher<BuildServerListener> buildServerListenerEventDi
             put("testHeader2", "testHeaderValue2");
         }};
         GlobalOpenTelemetry.resetForTest();
-        this.otelHelper = new OTELHelper(HEADERS, ENDPOINT);
-        this.buildListener = new TeamCityBuildListener(buildServerListenerEventDispatcher, otelHelper);
-    }
-
-    @Test
-    void illegalStateExceptionWhenHeadersEmpty(@Mock EventDispatcher<BuildServerListener> buildServerListenerEventDispatcher) {
```

Comment:
We now defer instantiation to build start time, so this test doesn't sense anymore